pitop/camera/core/cameras/usb_camera.py

```diff
@@ -10,7 +10,7 @@ def __init__(self, camera_index: int = 0):
         try:
             self.__camera = Camera(f"/dev/video{camera_index}")
         except CameraError:
-            raise IOError("Error opening camera. Make sure it's correctly connected via USB.") from None
+            raise IOError(f"Error opening camera {camera_index}. Make sure it's correctly connected via USB.") from None
```

Comment:
```suggestion
            raise IOError(f"Error opening camera {camera_index}. Make sure it's connected to the correct USB port.") from None
```

Reply:
Port is not relevant here - ID identifier is not appropriate.